pages/content/amp-dev/documentation/guides-and-tutorials/index.email.html

```diff
@@ -8,114 +8,199 @@
   path: /{locale}/documentation/guides-and-tutorials.email.html
 $order: 1
 $hidden: true
+
+flyout:
+  description: Get started with AMP
+  icon: guides-documents
+cms: !g.yaml /content/shared/data/cms.yaml
 ---
-{% do doc.styles.addCssFile('css/components/organisms/teaser-grid.css') %}
-<h1>Introduction to AMP for Email</h1>
 
-<p class="limit">AMP for Email allows you to create interactive emails using a subset of components from the AMPHTML library. AMP emails can update dynamically, allowing readers to view and interact with AMP components directly in the message. </p>
 
-{% do doc.styles.addCssFile('/css/components/organisms/benefits-overview.css') %}
-<div class="ap--benefits--list">
-  {% do doc.styles.addCssFile('/css/components/molecules/benefit.css') %}
-  <div class="ap-m-benefit">
-    <div class="ap-m-benefit-text">
-      <div class="ap-m-benefit-icon">
-        {% do doc.icons.useIcon('/icons/compass.svg') %}
-        <svg class="ap-m-benefit-icon">
-          <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#compass"></use>
-        </svg>
+{% do doc.styles.addCssFile('/css/components/organisms/format-cards.css') %}
+<section class="ap--format-cards">
+  <h1 class="ap-o-format-cards-text">Build emails with AMP!</h1>
```

Comment:
This doesn't seem to be ideally structured 😅 It would be nice if the introduction text and the headline would be in their own section.

Reply:
Done :)